Hangman/words/gr.json

```diff
@@ -1,5 +1,96 @@
 {
-	"words": ["javascript", "java", "python", "typescript", "κρεμάλα", "προγραμματιστής", "developer", "coder", "github", "google", "τερματικό", "μηχανή", "scripting", "διακομιστής", "πρόγραμμα", "σειρά", "bytecode", "εντολή", "λειτουργία", "κατασκευαστής", "διαφωνίες", "οπτικό", "μεταβλητή", "πηγή", "εκδότης", "microsoft", "βιβλιοθήκη", "εξάρτηση", "εκδοχή", "δοκιμαστικός", "πακέτο", "έγγραφα", "conditional", "επαναλήψεις", "βάση δεδομένων", "hacker", "δυαδικό", "bitwise", "δεδομένα", "αλγόριθμοι", "πελάτης", "localhost", "αντικείμενο", "τάξη", "πεδία", "ανάπτυξη", "boolean", "μέθοδοι", "παραγοντικός", "επιστροφή", "επανάληψη", "ενσάρκωση", "διακήρυξη", "τύπος", "στοίβα", "σωρός", "stackoverflow", "indentation", "regex", "μνήμη", "macros", "πληκτρολόγιο", "αυτοματισμός", "φάκελοι", "λεξικό", "symbolon", "κτίστης", "cryptex", "", "αριστούργημα", "πυραμίδα", "τελετουργικά", "νοητικός", "pythagorus", "ειλικρινής", "ναός", "πάπυρος", "εβραϊκά", "λατινικό", "μήλο", "βωμός", "crucifixion", "shaman", "darkness", "κρανίο", "vine", "ψυχή", "σύμβολα", "βιβλίο"],
+	"words": [
+		"javascript", 
+		"java", 
+		"python", 
+		"typescript", 
+		"κρεμάλα", 
+		"προγραμματιστής", 
+		"developer", 
+		"coder", 
+		"github", 
+		"google", 
+		"τερματικό", 
+		"μηχανή", 
+		"scripting", 
+		"διακομιστής", 
+		"πρόγραμμα", 
+		"σειρά", 
+		"bytecode", 
+		"εντολή", 
+		"λειτουργία", 
+		"κατασκευαστής", 
+		"διαφωνίες", 
+		"οπτικό", 
+		"μεταβλητή", 
+		"πηγή", 
+		"εκδότης", 
+		"microsoft", 
+		"βιβλιοθήκη", 
+		"εξάρτηση", 
+		"εκδοχή", 
+		"δοκιμαστικός", 
+		"πακέτο", 
+		"έγγραφα", 
+		"conditional", 
+		"επαναλήψεις", 
+		"βάση δεδομένων", 
+		"hacker", 
+		"δυαδικό", 
+		"bitwise", 
+		"δεδομένα", 
+		"αλγόριθμοι", 
+		"πελάτης", 
+		"localhost", 
+		"αντικείμενο", 
+		"τάξη", 
+		"πεδία", 
+		"ανάπτυξη", 
+		"boolean", 
+		"μέθοδοι",
+		"παραγοντικός",
+		"επιστροφή", 
+		"επανάληψη", 
+		"ενσάρκωση", 
+		"διακήρυξη", 
+		"τύπος", 
+		"στοίβα", 
+		"σωρός", 
+		"stackoverflow", 
+		"indentation",
+		"regex", 
+		"μνήμη",
+		"macros",
+		"πληκτρολόγιο", 
+		"αυτοματισμός", 
+		"φάκελοι", 
+		"λεξικό", "symbolon",
+		"κτίστης", 
+		"cryptex",
+		"", 
```

Comment:
There shouldn't be blank words

Reply:
Just used the ones from before, only put the words on in each line.